carHandler.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import json
+import os
+from uuid import uuid4
+
+with open('input.csv', 'r') as csv_file:
+    rows = csv.reader(csv_file, delimiter=';')
+    cars_data = list(rows)
+    keys = list(cars_data[0])
+    keys.insert(0, 'UUID')
+    keys = tuple(keys)
```

Comment:
Do you really need to create a list and insert into it and then to a tuple from it rather the just create a tuple with all the values at once?